classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader : student
```

Comment:
- [ ] great job setting up the attribute accessor for student 👍  however when implementing the requested changes by the previous CR you made a very small mistake which will cause it not to work, please remove the space between the full colon and the word 'student'